build-system/tasks/report-test-status.js

```diff
@@ -45,29 +44,32 @@ const TEST_TYPE_BUILD_TARGETS = new Map([
 ]);
 
 function inferTestType() {
-  if (IS_GULP_TEST) {
-    let type;
-    if (IS_UNIT) {
-      type = 'unit';
-    } else if (IS_INTEGRATION) {
-      type = 'integration';
-    } else {
-      return null;
-    }
-
-    if (IS_LOCAL_CHANGES) {
-      return `${type}/local-changes`;
-    } else if (IS_SAUCELABS) {
-      return `${type}/saucelabs`;
-    } else if (IS_SINGLE_PASS) {
-      return `${type}/single-pass`;
-    } else {
-      return `${type}/local`;
-    }
-  } else if (IS_GULP_E2E) {
+  if (IS_GULP_E2E) {
     return 'e2e/local';
   }
-  return null;
+
+  let type;
+  if (IS_GULP_UNIT) {
+    type = 'unit';
+  } else if (IS_GULP_INTEGRATION) {
+    type = 'integration';
+  } else {
```

Comment:
Should there be an `else if (IS_GULP_E2E)` here?

Reply:
No, it checks E2E above and returns

Reply:
:sheep: :smiley: